### PR TITLE
Feature/pin message

### PR DESCRIPTION
**Pull request by erschmidd**
Bildschirmfoto 2019-10-20 um 19 16 15

This example has only 1 pin reaction for testing.

### CODE REVIEW

File: config.json.example [hunk 1]
```diff
@@ -25,5 +25,9 @@
     "sleeplogReaction": "🛏️",
     "cotwVow": "⚔",
     "cotwReflection": "🛡"
+    "pinMessage": "📌"
+  },
+  "pinLimit": {
+  "pin": 5
```
Comment (Moesgaarda) [inline config.json.example]:
```suggestion
    "pin": 5
```

File: events/messageReactionAdd.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 const tosActions = require('../eventActions/tosActions');
+const pinAction = require('../eventActions/pinAction');
 
 module.exports = async (client, reaction, user) => {
 	// Handle reaction to the ToS message in ToS channel
 	tosActions.userAcceptsTOS(reaction, user, client);
+	pinAction.pinMessage(client, reaction);
```
Comment (Moesgaarda) [inline events/messageReactionAdd.js]: It would be nice with a comment above this, describing what it does 🚀

File: config.json.example [hunk 3]
```diff
@@ -25,5 +25,9 @@
...
   }
```
Comment (Moesgaarda) [inline config.json.example]: Is there a reason that this isn't just "pinLimit": 5? Does it need the pin child?

**Comment by Dendrobyte** [comment]
Fixes #43

**Pull request merged**